src/encoded/schemas/surgery.json

```diff
@@ -0,0 +1,153 @@
+{
+    "title": "Surgery",
+    "description": "Schema for surgery embedded in portal pages",
+    "id": "/profiles/surgery.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": [
+        "patient",
+        "date",
+        "surgery_type"
+    ],
+    "identifyingProperties": [
+        "uuid"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        },
+        {
+            "$ref": "mixins.json#/submitted"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "patient": {
+            "title": "Patient",
+            "description": "The patient the lab was conducted for.",
+            "type": "string",
+            "linkTo": "Patient"
+        },
+        "date": {
+            "title": "Surgery date",
+            "description": "Date of surgery.",
+            "comment": "Date can be submitted as YYYY-MM-DD.",
+            "type": "string",
+            "anyOf": [
+                {
+                    "format": "date"
+                }
+            ]
+        },
+        "surgery_type": {
+            "title": "Surgery procedure",
+            "description": "The surgery procedures conducted",
+            "type": "string",
+            "uniqueItems": true,
+            "items": {
+                "type": "string",
+                "enum": [
+                    "Ablation",
+                    "Nephrectomy",
+                    "Metastectomy"
+                ]
+            }
+        },
+        "nephrectomy_details": {
+            "title": "Nephrectomy details",
+            "description": "Further details of Neprhectomy",
+            "type": "object",
+            "permission": "import_items",
+            "additionalProperties": false,
+            "required": [
+                "type",
+                "approach",
+                "robotic_assist",
+                "location"
+            ],
+            "properties": {
+              "type": {
+                  "title": "Neprhectomy type",
+                  "type": "string",
+                  "enum": [
+                      "Partial",
+                      "Total"
+                  ]
+              },
+              "approach": {
+                  "title": "Neprhectomy Approach",
+                  "type": "string",
+                  "enum": [
+                      "Laparoscopic",
+                      "Open"
+                  ]
+              },
+              "robotic_assist": {
+                  "title": "Robotic Assit",
+                  "type": "boolean"
+              },
+              "location": {
```

Comment:
@SenseCode  Move Location to same level as Surgery Type. Add "Not Available" option for Surgery location.